### ISSUE TITLE
betteruptime_status_page_resource.widget_type resets itself when planning/applying

### ISSUE THREAD

**Issue by chronark** [BetterStackHQ/terraform-provider-better-uptime#64]
Hey guys, I'm running into issues with your terraform provider, the type of widget on the status randomly resets and I have traced it down to the terraform provider resetting it for some reason

here's a snippet of the tf code and output when I plan the changes

```tf
resource "betteruptime_status_page_resource" "landing_page" {
  public_name = "Landing Page"
  widget_type = "history"

  resource_id            = betteruptime_monitor.landing_page.id
  resource_type          = "Monitor"
  status_page_id         = betteruptime_status_page.status_page.id
  status_page_section_id = betteruptime_status_page_section.app.id
}
```

And then when I plan or apply it gets changed to `null`:
```
 # betteruptime_status_page_resource.landing_page will be updated in-place
  ~ resource "betteruptime_status_page_resource" "landing_page" {
      - history                = true -> null
        id                     = "8308274"
        # (9 unchanged attributes hidden)
    }
```
    
    
```
Terraform v1.6.5 on darwin_arm64
+ provider registry.terraform.io/betterstackhq/better-uptime v0.4.0
```

**Comment by PetrHeinz** [comment]
Hello @chronark, thanks for reporting it and for your patience on this one 🙌

Specifying the field as Optional and Computed will make sure the API-returned values are always respected. I've proposed a fix in https://github.com/BetterStackHQ/terraform-provider-better-uptime/pull/96 and the issue should be addressed in `v0.10.0`.

Your configuration should produce no more random resets like that after upgrading. Apologies for the issue 🙏

**Comment by PetrHeinz** [comment]
Just released `v0.10.0` which fixes the issue with optional field being reset 🙌
